doc/update.md

```diff
@@ -25,18 +44,18 @@ The following arguments are available:
 
 ## Winget-Create update Command flow
 The update command allows you to quickly and easily update your manifest and submit a PR. **Winget-Create** can be integrated into your build pipeline to respond to your publishing of your installer.
-1) publish your installer to known URL
+1) publish your installer to known URL(s)
 2) call the WingetCreateCLI.exe
 
-`WingetCreateCLI.exe update <PackageIdentifier> --url <InstallerUrl> --token <token> --version <version>`
+`WingetCreateCLI.exe update <PackageIdentifier> --urls <InstallerUrls> --token <token> --version <version>`
 
 ### PackageIdentifier  
 
 The first action **Winget-Create** will do is to download the metadata associated with the **PackageIdentifier**.
 
 **Winget-Create** will then replace the following manifest values with the values provided on the command line:
```

Comment:
This list is pretty limited compared to what we do now. I might just remove it altogether, as it'll be hard to keep it up to date.

Reply:
I agree. I decided to remove both the "Winget-Create update Command flow" and the "PackageIdentifier" sections as they're not particularly useful and out of date. There's better information about using wingetcreate in a CI/CD scenario in the readme doc.